base/essentials.jl

```diff
@@ -623,6 +623,10 @@ getindex(v::SimpleVector, I::AbstractArray) = Core.svec(Any[ v[i] for i in I ]..
 Test whether the given array has a value associated with index `i`. Return `false`
 if the index is out of bounds, or has an undefined reference.
 
+    isassigned(ref) -> Bool
+
+Test whether the given [`Ref`](@ref) has been associated with a value. This is always true for a [`Ref`](@ref) for a bitstype. Return `false` if the reference is undefined.
+
```

Comment:
Added `isassigned(ref)` documentation with cross reference to `Ref`

Reply:
I think this (and the example) should add to the docstring of `isassigned(::RefValue)` instead. You can use `methods(isassigned)` (or equivalent) to find where it is.

Reply:
Done in 2849f95